CHANGELOG.md

```diff
@@ -2,310 +2,504 @@
 
 This changelog includes all versions and major variants of the mod going all the way back to March 2015, just 4 days after Cities: Skylines was released!
 
-> **Legend:**
->  
+<details><summary>Evolution of TM:PE (click to show/hide)</summary>
+
 > * **C:SL** = Cities: Skylines game updates (for reference)
-> * **TM:PE** = Traffic Manager: President Edition
->     * TM:PE v11 STABLE - TM:PE versions 11.0 and above (stable releases)
->     * TM:PE v11 LABS - TM:PE versions 11.0 and above (test releases)
->     * TM:PE v11 ALPHA - TM:PE versions 11.0-alpha1 to 11.0-alpha12 (later renamed to TM:PE v11 LABS)
->     * TM:PE LABS - TM:PE versions 10.14 to 10.21.1 (later renamed to TM:PE v11 STABLE)
->     * TM:PE STABLE - TM:PE versions 10.14 to 10.20
-> * **TPP2** = Traffic++ V2
-> * **TM:IAI** = Traffic Manager + Improved AI
-> * **TMPlus** = Traffic Manager Plus
-> * **TPP:AI** - Traffic++ Improved AI
-> * **TPP** = Traffic++
-> * **TM** = Traffic Manager
-> * **CSLT** = Cities Skylines Traffic (later renamed to Traffic++)
 > * **TLM** = Traffic Lights Manager (later renamed to Traffic Manager)
+> * **CSLT** = Cities Skylines Traffic (later renamed to Traffic++)
+> * **TM** = Traffic Manager
+> * **TPP** = Traffic++
+> * **TPP:AI** - Traffic++ Improved AI
+> * **TMPlus** = Traffic Manager Plus
+> * **TM:IAI** = Traffic Manager + Improved AI
+> * **TPP2** = Traffic++ V2
+> * **TM:PE** = `Traffic Manager: President Edition` versions 1 to 10.14
+> * **TM:PE LABS** - TM:PE versions 10.14 to 10.21.1 (later renamed to TM:PE v11 STABLE)
+> * **TM:PE STABLE** - TM:PE versions 10.14 to 10.20
+> * **TM:PE v11 ALPHA** - TM:PE versions 11.0-alpha1 to 11.0-alpha12 (later renamed to TM:PE v11 LABS)
+> * **TM:PE v11 LABS** - TM:PE versions 11.0 to 11.5.0 (test releases)
+> * **[TM:PE v11 STABLE](https://steamcommunity.com/sharedfiles/filedetails/?id=1637663252) - TM:PE versions 11.0 and above (stable releases)**
+> * **[TM:PE v11 TEST](https://steamcommunity.com/sharedfiles/filedetails/?id=2489276785) - TM:PE versions 11.6.0 and above (test releases)**
 >  
 > Date format: dd/mm/yyyy
 
+</details>
+
+
+#### TM:PE 11.6.3.0 TEST, Soon
+
+- [New] What's New panel #118 #1284 (krzychu124, aubergine18)
+- [Mod] Incompatible: `Traffic++ V2` - ancient mod, breaks game #1287 (aubergine18)
+- [Fixed] App mode detection #1290 #1289 #1286 (krzychu124)
+- [Fixed] Versioned atlas to ensure textures load properly #1281 (kianzarrin)
+- [Updated] Speed Limits manager for asset editor and mods #1277 #1285 (kianzarrin, krzychu124)
+
+#### TM:PE 11.6.2.0 TEST, 08/01/2022
+
+- [New] Speed Limits UI Overhaul #1168 #389 (kvakvs)
+- [New] Road sign icon themes for multiple countries #1260 #1251 #1215 (Coreybpa, kvakvs, aubergine18)
+- [New] Dynamic loading of road sign icon themes #1215 (kvakvs)
+- [New] Default speeds apply to all elevations #1245 #1233 #1226 #1221 #1168 (kvakvs, kianzarrin)
```

Comment:
This is not a new feature but more like a fix to # 1168

Reply:
Comparing to <11.6 it's something new as previously you've had to apply default speed to each elevation manually iirc